src/pages/workspaces/workspace/Analyses.js

```diff
@@ -258,11 +257,6 @@ const Analyses = _.flow(
     withErrorReporting('Error loading analyses'),
     Utils.withBusyState(setBusy)
   )(async () => {
```

Comment:
This may cause a change in behavior that might not be desirable with regard to "requester pays" workspaces. The call in `WorkspaceContainer` to obtain the location/locationType does **not** use `withRequestPaysHandler` (nor do we want it to, as that would force the user to select a billing workspace whenever any part of the Workspace was rendered); in the case of a requester pays workspace, a 400 error will result and location/locationType will not be populated.

When you remove these extra calls to `checkBucketLocation`, please make sure that there is still handling in each location that will trigger the "requester pays" logic appropriately, and that the values then populate after the user selects a workspace to bill to.